src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -405,6 +407,22 @@ void AutomatedRssDownloader::on_removeRuleBtn_clicked()
         RSS::AutoDownloader::instance()->removeRule(item->text());
 }
 
+void AutomatedRssDownloader::on_addCategoryBtn_clicked()
+{
+    QString prevCategoryName = m_ui->comboCategory->currentText();
+    QString newCategoryName = TorrentCategoryDialog::createCategory(this);
+
+    if (newCategoryName.isEmpty())
+    {
+        m_ui->comboCategory->setCurrentText(prevCategoryName);
```

Comment:
What for? It's already current.

Reply:
Somehow when I was working on I thought this was necessary. There is no need to set it back to the original.